content/content-strategy/cms/purpose-cms/section.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: text/textblock
+---
+The main purpose of a CMS is to manage content in one place. Your CMS is a support tool for your [content strategy](https://guides.service.gov.au/content-strategy/). It should ensure that content is available in the most useful way possible.
```

Comment:
Should be a relative path, *not* the full URL that includes `https://guides.service...`

Reply:
AFAIK the link should just be `/content-strategy/` I'd double check the link works before commiting ;-)

Reply:
Steve. Fixed